WebRoot/js/zimbraAdmin/servers/model/ZaServer.js

```diff
@@ -154,6 +154,7 @@ ZaServer.A_VolumeType = "type";
 ZaServer.A_CurrentIndexVolumeId = "current_index_volume_id";
 ZaServer.A_CurrentMsgVolumeId = "current_msg_volume_id";
 ZaServer.A_isCurrent = "isCurrent";
+ZaServer.LBL_VolumeRootPath = ZaMsg.VM_VolumeRootPath;
```

Comment:
we can remove this line

Reply:
we would need it for properly overwriting the value from the zimlet, unless there's another way of setting the value of ZaMsg.properties from the zimlet itself.